google/cloud/internal/grpc_request_metadata.cc

```diff
@@ -66,10 +68,15 @@ void AddContextMetadata(grpc::ClientContext const& context,
 
 }  // namespace
 
-RpcMetadata GetRequestMetadataFromContext(grpc::ClientContext const& context) {
+RpcMetadata GetRequestMetadataFromContext(grpc::ClientContext const& context,
+                                          internal::ErrorOrigin error_origin) {
   RpcMetadata metadata;
   AddContextMetadata(context, metadata);
-  AddServerRequestMetadata(context, metadata);
+  // If there is a client originating error, the request never made it to the
```

Comment:
Perhaps `s/is a client originating error/was a client-originated error/`.

Reply:
Done.